index.html

```diff
@@ -12,14 +12,21 @@
     />
   </head>
   <body>
-    <header class="toolbar-container">
-      <a class="logo" href="#">My Logo</a>
-      <nav class="nav-container">
+    <header>
+      <nav class="nav-container toolbar-container"> 
+        <a class="logo" href="#">My Logo</a>  
         <i class="fa fa-solid fa-bars"></i>
+        <ul class="menu-bar">
+          <li class="nav">Portfolio</li>
+          <li class="nav">About</li>
+          <li class="nav">Contact</li>
+        </ul>
```

Comment:
- Good job wrapping this in a `ul` tag and also in an `li` tag 👍 , but I'd suggest you wrap then in an `a` tag first before wrapping each of them in an `li` tag. This would help for easy navigation around your site, and aslo help to improve the accessibility of your site 👍 😄